src/frontend/src/redux/records/types.ts

```diff
@@ -22,18 +8,15 @@ export interface Record {
   caseBalance: string;
 }
 
-export interface RecordsState {
+export const LOAD_SEARCH_RECORDS = 'LOAD_SEARCH_RECORDS';
+
+export interface SearchRecordState {
   records: Record[];
 }
 
-// These constants are used as the 'type' field in Redux actions.
-export const LOAD_RECORDS = 'LOAD_RECORDS';
-
-interface LoadRecordsAction {
-  type: typeof LOAD_RECORDS;
-  records: Record[];
+interface SearchRecordsAction {
+  type: string;
```

Comment:
Can you keep this as the typeof the constant used to signal this action? That way makes it much clearer for someone reading the code to see the relationship.

Reply:
Done